src/ProjectOrigin.WalletSystem.Server/Activities/SendInformationToReceiverWalletActivity.cs

```diff
@@ -80,31 +82,31 @@ private async Task<ExecutionResult> SendOverGrpcToExternalWallet(ExecuteContext<
         }
     }
 
-    private async Task<ExecutionResult> InsertIntoLocalWallet(ExecuteContext<SendInformationToReceiverWalletArgument> context, Slice newSlice, DepositEndpoint receiverDepositEndpoint)
+    private async Task<ExecutionResult> InsertIntoLocalWallet(ExecuteContext<SendInformationToReceiverWalletArgument> context, OutboxSlice newSlice, OutboxEndpoint outboxEndpoint)
     {
         _logger.LogTrace("Receiver is local.");
```

Comment:
Not changed in this PR, but is log statement correct?

Reply:
Changed to LogDebug, Trace would probably not currently be used.